.pipelines/nuget-publishing.yml

```diff
@@ -36,17 +36,17 @@ parameters:
 - name: ort_version
   displayName: 'OnnxRuntime version'
   type: string
-  default: '1.18.0-dev-20240426-1256-b842effa29'
+  default: '1.19.0-dev-20240805-1630-ee2fe87e2d'
```

Comment:
Note that these versions will be written to the final nuspec as a dependency. We should probably only use official ORT release as the dependency.

Reply:
I think when the 1.19.0 stable is published, we can update the version here.

Reply:
Come to think about it, the current dependency has strict version requirement like this:

```xml
<dependency id="Microsoft.ML.OnnxRuntime.Gpu" version="1.19.0-dev-20240805-1630-ee2fe87e2d" />
```

Maybe we can relax it a bit?

Reference: https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort

Reply:
@natke This is how the dependency is added, as I mentioned in your doc PR.